hw/ip_templates/ac_range_check/rtl/ac_range_check.sv.tpl

```diff
@@ -0,0 +1,90 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+module ${module_instance_name} 
+  import tlul_pkg::*;
+  import ${module_instance_name}_reg_pkg::*; 
+#(
+) (
+  input                                             clk_i,
+  input                                             rst_ni,
+  input                                             rst_shadowed_ni,
```

Comment:
Suggest `input logic ` here. I know we don't consistently do this so far, but for new code we might as well